src/components/dealer/dealer.tsx

```diff
@@ -75,7 +87,7 @@ export const Dealer = (props: DealerProps) => {
     return (
         <div className={styles.dealer}>
             <div className={styles.name}>DEALER</div>
-            <HandOfCards blackjackHand={getCardsToDisplay()} />
```

Comment:
remove the `getCardsToDisplay()` function as it is unused now

Reply:
perhaps due to time-skip in working but what replaces this?